dirty_cat/datasets/fetching.py

```diff
@@ -600,3 +604,52 @@ def fetch_drug_directory(
         },
         load_dataframe=load_dataframe,
     )
+
+
+def fetch_world_bank_data(data_code: str, indicator: str,
+                          load_dataframe: bool = True,
+                          ) -> Union[DatasetAll, DatasetInfoOnly]:
```

Comment:
I tried to reuse them as best as I could, finally passed all the tests. I will take a closer look in future PR's and discuss with @LilianBoulard if it can be cleaner.